vm/vm.go

```diff
@@ -158,6 +165,9 @@ func New(
 	// Set defaults
 	vm.builder = builder.NewTime(vm)
 	vm.gossiper = txGossiper
+	vm.heightKey = []byte("height")
+	vm.timestampKey = []byte("timestamp")
+	vm.feeKey = []byte("fee")
```

Comment:
todo we might introduce keys that conflict w/ the controller's predefined keys so it would be good to have a way to detect conflicts in the implementation side during initialization

Reply:
Alternatively, we could make the default to prefix the HyperSDK metadata and have a separate prefix for standard VM implementations portion of state.

Reply:
Another idea is to have an entirely separate state db for hypersdk defined state vs for state the controller defines